src/app/header.html

```diff
@@ -108,7 +108,7 @@
                 <ul class="dropdown-menu">
                     <li>
                         <a target="_blank"
-                            href="https://omero-guides.readthedocs.io/projects/iviewer/">
+                            href="https://omero-guides.readthedocs.io/en/latest/iviewer/docs/index.html">
```

Comment:
Could optionally remove the `index.html`, I think it makes the URL look a bit nicer but not essential

Reply:
:+1: